src/zeroD/ReactorNet.cpp

```diff
@@ -269,16 +269,17 @@ void ReactorNet::eval(doublereal t, doublereal* y,
                     "eval(double t, double* ydot)");
                 m_have_deprecated_eval[n] = true;
             } catch (NotImplementedError& err) {
+                (void)err; // suppress compiler warning
```

Comment:
The better way to avoid this compiler warning is to just write the `catch` statement without giving the exception a name, e.g.:
```c++
} catch (NotImplementedError&) {
```